### PR TITLE
Finish testing effect classes

### PR DESCRIPTION
**Pull request by Huy-Ngo**
This PR will finalize the tests for `ReverbEffect` and `ChorusEffect`.

### CODE REVIEW

**Comment by McSinyx** [comment]
~~seems to me that long as the type is correct, openal won't complain~~

**Comment by McSinyx** [comment]
Don't wait for GH-88 to be fix to finish this though.  You can always `mark` tests with `xfail`.

**Comment by Huy-Ngo** [comment]
Yea I know, I just have several other things to do.

File: tests/unit/test_effect.py [hunk 1]
```diff
@@ -19,9 +19,11 @@
 
 """This pytest module verifies environmental effects."""
 
-from palace import BaseEffect, ReverbEffect, Source
+from palace import BaseEffect, ChorusEffect, ReverbEffect, Source
 from pytest import raises
```
Comment (McSinyx) [inline tests/unit/test_effect.py]: Import `mark` here and use `@mark.xfail` instead.  BTW it'd be nice if you can squash reverb and chorus tests into 2 separate commits (xfail mark can be in the latter one).

**Review by McSinyx — APPROVED** [review]

**Comment by McSinyx** [comment]
Should have been rebase lol.

**Pull request merged**